tests/test_fbx.py

```diff
@@ -79,3 +81,19 @@ def test_derivative(modelJ0023, toasJ0023):
             assert np.nanmax(relative_diff) < tol, msg
         else:
             continue
+
+
+def test_summary_FB():
+    m = mb.get_model(os.path.join(datadir, parJ0023ell1))
```

Comment:
Wouldn't this code be very much simpler if you just used `get_model_and_toas`? Like couldn't you drop all that stuff about posvels and planet ephem?

Reply:
Yes. :)